src/libraries/System.IO/tests/IndentedTextWriter.cs

```diff
@@ -178,5 +465,330 @@ public static async Task Writes_ProducesExpectedOutput(string newline)
+                yield return CreateParameters(x => x.WriteLine(12345), $"{12345}{NewLine}");
+                yield return CreateParameters(x => x.WriteLine(0xDEADBEEFBADF00DL), $"{0xDEADBEEFBADF00DL}{NewLine}");
+                yield return CreateParameters(x => x.WriteLine(new object()), $"{new object()}{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello World"), $"Hello World{NewLine}");
+                yield return CreateParameters(x => x.WriteLine(0xDEADBEEF), $"{0xDEADBEEF}{NewLine}");
+                yield return CreateParameters(x => x.WriteLine(0xDEADBEEFBAADF00DUL), $"{0xDEADBEEFBAADF00DUL}{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello {0} World", "Digital"), $"Hello Digital World{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello {0} {1} World", "Dot", "NET"), $"Hello Dot NET World{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello {0} {1} World{2}", "Dot", "NET", "!!"), $"Hello Dot NET World!!{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello World".ToCharArray(), 6, 5), $"World{NewLine}");
+                yield return CreateParameters(x => x.WriteLine("Hello {0} {1} {2} World{3}", "Digital", "Dot", "NET", "!!"), $"Hello Digital Dot NET World!!{NewLine}");
+            }
+        }
+
+        public static IEnumerable<object[]> WriteAsync_MemberData
+        {
+            get
+            {
+                object[] CreateParameters(Func<IndentedTextWriter, Task> callWriteAsync, string expected)
+                {
+                    return new object[] { callWriteAsync, expected };
+                }
+
+                yield return CreateParameters(x => x.WriteAsync('c'), "c");
+                yield return CreateParameters(x => x.WriteAsync("Hello World".ToCharArray(), 6, 5), "World");
+                yield return CreateParameters(x => x.WriteAsync("Hello World"), "Hello World");
+            }
+        }
+
+        public static IEnumerable<object[]> WriteLineAsync_MemberData
+        {
+            get
+            {
+                object[] CreateParameters(Func<IndentedTextWriter, Task> callWriteLineAsync, string expected)
+                {
+                    return new object[] { callWriteLineAsync, expected };
+                }
+
+                yield return CreateParameters(x => x.WriteLineAsync(), NewLine);
+                yield return CreateParameters(x => x.WriteLineAsync('c'), $"c{NewLine}");
+                yield return CreateParameters(x => x.WriteLineAsync("Hello World".ToCharArray(), 6, 5), $"World{NewLine}");
+                yield return CreateParameters(x => x.WriteLineAsync("Hello World"), $"Hello World{NewLine}");
+            }
+        }
+
+        private const string TabString = "   ";
+        private const string NewLine = "\n";
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Did you have to add this because the CI failing in .NET Framework? If you didn't see any failures, maybe the attribute is not needed.

Reply:
I just ran the tests without it and they passed. Not sure why I added them, it was a couple weeks ago. I'll remove them.

Reply:
Well, CodeDome test project had .Net framework target `net461`, i guess it was failing before moving tests to System.IO https://github.com/dotnet/runtime/blob/32d664d10ce92f6535606b4ca658d0e9cfd754a4/src/libraries/System.CodeDom/tests/System.CodeDom.Tests.csproj#L4

now Syste.IO doesn't have .Net framework target